web/packages/teleport/src/services/apps/types.ts

```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 
+import { ResourceIconName } from 'design/ResourceIcon';
+
 import { ResourceLabel } from 'teleport/services/agents';
 
+type GuessedAppType = ResourceIconName;
```

Comment:
i would leave the type as is imo (ResourceIconName), unless we narrow the options down to just app related icons?